modules/Template/Dataset.yaml

```
@@ -0,0 +1,132 @@
+#######################################
+# Portal Dataset
+#######################################
+
+classes:
+  Dataset:
+    description: Model for dataset on the AMP-ALS portal
+    notes:
+    - There will be mappings available to this target schema from various source schemas stored in Core Models or under mappings/*
+    attributes:
+      title:
+        title: Title
+        range: string
+        required: true
+      description:
+        title: Description
+        range: string
+        description: Blurb for the dataset; should be no more than 500 characters.
+        required: false
+      alternateName:
+        title: Alternate Name
+        description: An alternate name that can be used for search and discovery improvement.
+        range: string
+        required: false
+      creator:
+        title: Creator
+        range: string
+        description: Properly formatted name of the organization or person that is creator of the dataset (e.g. "NF-OSI" or "Robert Allaway"), not an id. Most preferred is the project PI or data lead who's responsible for its conception and initial content creation.
+        multivalued: true
+        required: true
+      contributor:
+        title: Contributor(s)
+        range: string
+        description: Institution or person responsible for collecting and managing the files and records in the dataset.
+        multivalued: true
+        required: false
+      # studyId:
+      #  title: Study ID
+      #  range: string
+      #  description: Identifier for the study (project) from which the dataset was derived.
+      #  required: true
+      measurementTechnique:
+        title: Measurement Technique
+        range: AssayEnum
+        multivalued: true
+        description: What's used to generate data in dataset, typically referring to the assay. Omit if not applicable.
+      keywords:
+        title: Keywords
+        range: string
+        multivalued: true
+        required: true
+      subject:
+        title: Subject
+        range: string
+        description: Applicable subject term(s) for dataset cataloging; use the Library of Congress Subject Headings (LCSH) scheme.
+        multivalued: true
+        required: false
+      #dataType:
+      #  title: Data Type
+      #  range: Data
+      #  description: Reflects the data types within the dataset.
+      #  multivalued: true
+      #  required: true
+      species:
+        title: Species
+        range: SpeciesEnum
+        description: Species of the organism(s) from which the data were generated. Omit property if not applicable, such as for data like compounds or other non-biological data.
+        multivalued: true
+        required: false
+      funder:
+        title: Funder
+        range: string
+        multivalued: true
+        required: true
+      #individualCount:
```

```suggestion
      individualCount:
```

Suggest leaving this in: my understanding is that we'll have this information, at least for the RDCA-DAP datasets